tac/project-updates/2025/2025-Q3-Hyperledger-Solang.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: 2025 Q3 Hyperledger Solang
+parent: Project Updates
+grand_parent: LF Decentralized Trust TAC
+nav_exclude: true
+---
+
```

Comment:
```suggestion
```

Please note that we have switched to using Material for mkdocs. Please remove this front matter and add the file to the `Project Updates` section of the `mkdocs.yml` file in the root directory.